### ISSUE TITLE
Cannot perform relative import

### ISSUE THREAD

**Issue by bbc2** [tomv564/pyls-mypy#36]
Some relative imports cause errors. Here's an example:

Directory structure:

```
foo/
├── bar.py
├── baz.py
└── __init__.py
```

`bar.py`:

```python
a = 17
```

`baz.py`:

```python
from . import bar

a = bar.a
```

Runtime test:

```python
> python
Python 3.8.0 (default, Oct 23 2019, 18:51:26) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import foo.baz
>>> foo.baz.a
17
```

Mypy test:

```bash
> mypy foo
Success: no issues found in 3 source files
```

But when `pyls-mypy` is installed, the language server complains about the relative import in `baz.py`:

> No parent module -- cannot perform relative import

If I uninstall this extension, the error disappears and the rest of the language server works as expected but without Mypy messages.

**Comment by tomv564** [comment]
This can probably be fixed by turning `live_mode` setting off (see example config in readme) - the as-you-type checker doesn't know where a given chunk of code lives in the filesystem.

**Comment by bbc2** [comment]
Ah, that seems fair.  Turning off live mode indeed fixes the issue.

**Comment by tomv564** [comment]
Thanks for confirming! I'm curious if using the mypy daemon will fix it.
